BD-4676: Copying Canvases across workspaces GA (#9619)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/messaging_fundamentals/copying_across_workspaces.md

---
nav_title: Copying Across Workspaces
article_title: Copying Across Workspaces
page_order: 4.5
alias: "/copying_to_workspaces/"
page_type: reference
description: "This reference article provides an overview of how to copy campaigns and Canvases to different workspaces."
tool:
    - Campaigns
    - Canvas
---

# Copying campaigns and Canvases across workspaces

> Copying campaigns across workspaces lets you jumpstart your message composition by starting with a copy of a campaign in a different workspace. This page covers how to copy campaigns to different workspaces and lists what is and isn't copied over.

> 
> ## How it works

Reviewer: ```suggestion
```

This technically isn't describing the "behind the scenes" of how it works, so I think we could remove the header and let the blank space be.

{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

{% endtab %}
{% endtabs %}

### Content Blocks

{% tabs local %}
{% tab campaigns %}

| Copied | Omitted |
|---|---|
| Name | Link aliasing |
| Description | API keys | 
| Content | Territories | 
| HTML and drag-and-drop content | Tags | 
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

{% endtab %}
{% tab canvas %}

| Copied | Omitted |
|---|---|
| Name | Link aliasing |
| Description | API keys | 
| Content | Territories | 
| HTML and drag-and-drop content | Tags | 
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

{% endtab %}
{% endtabs %}

### SMS message variation

{% tabs local %}
{% tab campaigns %}

| Copied | Omitted |
|---|---|
| Body | Messaging service |
| Link shortening | VCF media items | 
| Click tracking |  | 
| Media items |  | 
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

{% endtab %}
{% tab canvas %}

| Copied | Omitted |
|---|---|
| Body | Messaging service |
| Link shortening | VCF media items | 
| Click tracking |  | 
| Media items |  | 
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

{% endtab %}
{% endtabs %}

## Copying messages that contain Liquid

> For message bodies that include Liquid references, the references are copied over to the destination workspace, but they may not function as expected. This means if a Canvas from Workspace A is copied to Workspace B, then Workspace B can't reference Workspace A's details, including Liquid references. For example, fields like trigger actions and audience filters aren't copied over.

Reviewer: ```suggestion
Liquid references within message bodies are copied over to the destination workspace, but the references may not function as expected. This means if a Canvas from Workspace A is copied to Workspace B, then Workspace B can't reference Workspace A's details, including Liquid references. For example, fields like trigger actions and audience filters aren't copied over.
```

> Note the following Liquid references with dependencies when copying campaigns and Canvases across workspaces:

Reviewer: ```suggestion
Keep track of the following Liquid references with dependencies when copying campaigns and Canvases across workspaces, as they may not function as expected:
```
Optional!
